build_heatmap_data.py

```diff
@@ -0,0 +1,107 @@
+"""
+Gregory Way 2017
+PanCancer Classifier
+build_heatmap_data.py
+
+Usage: Run in command line with required command argument:
+
+        python build_heatmap_data.py --genes $GENES
+
+Where GENES is a comma separated string. There are also optional arguments:
+
+    --copy_number       store_true: supplement copy number to define Y
+    --remove_hyper      store_true: remove hypermutated samples
+
+Output:
+.tsv file of gene events per TCGA disease types saved in the tables/ folder
+"""
+
+import os
+import warnings
+import pandas as pd
+import argparse
+from tcga_util import integrage_copy_number
+
+parser = argparse.ArgumentParser()
+parser.add_argument('-g', '--genes',
+                    help='Comma separated string of HUGO gene symbols')
+parser.add_argument('-u', '--copy_number', action='store_true',
+                    help='Supplement Y matrix with copy number events')
+parser.add_argument('-v', '--remove_hyper', action='store_true',
+                    help='Remove hypermutated samples')
+parser.add_argument('-f', '--alt_folder', default='Auto',
+                    help='location to save')
+args = parser.parse_args()
+
+# Load command arguments
+genes = args.genes.split(',')
+copy_number = args.copy_number
+remove_hyper = args.remove_hyper
+alt_folder = args.alt_folder
+
+if alt_folder == 'Auto':
+    base_folder = 'tables'
+else:
+    base_folder = os.path.join(alt_folder, 'tables')
+
+if not os.path.exists(base_folder):
+    os.makedirs(base_folder)
+
+gene_output_file = os.path.join(base_folder, '{}_mutations_heatmap_data.csv'
+                                .format(args.genes.replace(',', '_')))
+
+# Load Datasets
+expr_file = os.path.join('data', 'pancan_rnaseq_freeze.tsv')
```

Comment:
I think this is a great point that points to an even bigger issue. I've outlined it here in #8 . I think that I'd rather deal with data being in the right spot once all of the data is made public.